src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
> Yeah, like the "to ignore or not ignore the composer.lock"

That's not trivial though. That actually makes a difference, unlike how you write integer in a comment, lol.